applications/tari_dan_wallet_web_ui/src/routes/Wallet/Components/Accounts.tsx

```diff
@@ -113,11 +118,12 @@ function Accounts() {
   useEffect(() => {
     loadAccounts();
   }, []);
-  if (error) {
-    return <Error component="Accounts" message={error} />;
-  }
+
   return (
     <>
+      {error ? (
+          <Alert severity="error">{error}</Alert>
+      ) : ( <span></span>) }
```

Comment:
I think you can have `null` instead of empty `<span>` tag

Reply:
thanks